src/layers/graph-layer/graph-layer.js

```diff
@@ -267,12 +306,13 @@ export default class GraphLayer extends Layer {
         parameters: {
           depthTest: this.config.columns.altitude?.fieldIdx > -1
         },
-        lineWidthUnits: 'pixels',
+        sizeUnits: 'pixels',
         updateTriggers,
         extensions,
 
         // IconLayer stuff
-        getSize: 32
+        getSize: 32,
+        getColor: data.getIconColor
```

Comment:
so this is re-coloring the png icons according the color ranges? 😮

Reply:
Indeed!, if you look at the top of the file i added a `mask: true` setting to the icons. This tells kepler to use them as a mask and not as an actual PNG.